src/index.js

```diff
@@ -94,6 +102,18 @@ export default (inputs) => {
     }
 
     try {
+      if (schema) {
```

Comment:
If the schema is `true`, are we still checking the others (headers, query ...etc)?

Reply:
check workers APS-38 MR please. schema contains Joi object. In general its already prepared Joi object and in validator no need to call Joi.object function